### PR TITLE
TJvSpinButton: Scale button width according to PixelsPerInch 

### PR DESCRIPTION
**Pull request by MHumm**
of the monitor the button is on as an attempt to fix Mantis issue 6504
[http://issuetracker.delphi-jedi.org/view.php?id=6504](url)

I hope the fix is good enough

### CODE REVIEW

File: jvcl/run/JvSpin.pas [hunk 1]
```diff
@@ -1905,6 +1930,13 @@ constructor TJvUpDown.Create(AOwner: TComponent);
   Position := 0;
 end;
 
+function TJvUpDown.GetDefaultButtonWidth: Integer;
+begin
+  result := DefBtnWidth(Screen.MonitorFromRect(
+                          TRect.Create(Left, Top,
```
Comment (obones) [inline jvcl/run/JvSpin.pas]: Please do not block align, it's really hard to follow.

File: jvcl/run/JvSpin.pas [hunk 2]
```diff
@@ -1905,6 +1930,13 @@ constructor TJvUpDown.Create(AOwner: TComponent);
...
+                                       Left + Width, Top + Height)).PixelsPerInch);
```
Comment (obones) [inline jvcl/run/JvSpin.pas]: `PixelsPerInch` appeared with RS 10 Seattle. Before that it's only available directly on the `TScreen` object
So the three lines should be rewritten like this:

    Result := DefBtnWidth(Screen.
      {$IFDEF RTL300_UP}
      MonitorFromRect(TRect.Create(Left, Top, Left + Width, Top + Height)).
      {$ENDIF RTL300_UP}
      PixelsPerInch;

**Comment by MHumm** [comment]
I created #113 with the suggested changes so I close this one and hope #113 is good enough.
